model/fn-execution/src/main/resources/org/apache/beam/model/fnexecution/v1/standard_coders.yaml

```diff
@@ -28,6 +28,23 @@
 #             one of a few standard JSON types such as numbers, strings, dicts that map naturally
 #             to the type encoded by the coder.
 #
+# Java code snippet to generate example bytes:
+#   Coder<Timer<String>> coder = Timer.Coder.of(StringUtf8Coder.of(), GlobalWindow.Coder.INSTANCE);
+#   Instant now = new Instant(1000L);
+#   Timer<String> timer = Timer.of(
+#     "key",
+#     "tag",
+#     now,
+#     now,
+#     Collections.singletonList(GlobalWindow.INSTANCE),
+#     PaneInfo.NO_FIRING);
+#   byte[] byets = CoderUtils.encodeToByteArray(coder, timer);
+#   String str = new String(byets, java.nio.charset.StandardCharsets.ISO_8859_1);
```

Comment:
Kind of sure since it works for me. And I found usage here: https://github.com/apache/beam/blob/03c8544227530eb01dd0c64335e404ea387ee3a8/runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/CommonCoderTest.java#L190

Reply:
sg